lib/boolean.js

```diff
@@ -41,6 +54,50 @@ internals.Boolean = class extends Any {
         result.errors = (typeof result.value === 'boolean') ? null : this.createError('boolean.base', null, state, options);
         return result;
     }
+
+    truthy(truthyValue) {
+
+        let truthySet = [];
+
+        if (Array.isArray(truthyValue)) {
+            truthySet = truthyValue.map((value) => {
+
+                Hoek.assert(typeof value === 'string', 'truthy value must be a string');
+
+                return value.toLowerCase();
+            });
+        }
+        else {
```

Comment:
Why not just cast the value to an array if it isn't one currently? That way you don't have the code repeated. So before line 62 something like: `truthyValue = Array.isArray(truthyValue) ? truthyValue : [];`